testsuite-real/scaffold

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eu
+set -x
+# TODO check root, explain that snapper needs it
+
+CFG=testsuite
+MNT=/"$CFG"
+#CFG=scratch2
+#MNT=~/"$CFG"-btrfs
+IMG="$MNT".img
+LOOP=$(losetup --find)
+
+setup() {
+    dd if=/dev/zero of="$IMG" bs=1M count=110
```

Comment:
```truncate``` can be used instead of dd. Should be faster and use less disk space.

Reply:
OK, `truncate -s 110M` works too, thanks!